docs/developer_usage.rst

```diff
@@ -37,49 +37,51 @@ Checklist
 
 For the new tool you are trying to develop, there are a few criteria to satisfy
 
-1. Create a subdirectory for the tool group inside the ``fre/`` directory; i.e. ``fre/<subtool>``
+1. Create a subdirectory for the tool group inside the ``fre/`` directory; i.e. ``fre/<tool>``
 
-2. Add an ``__init__.py`` inside of ``fre/<subtool>`` 
+2. Add an ``__init__.py`` inside of ``fre/<tool>`` 
 
-* typically this file should be empty, but it depends on the ``<subtool>``'s needs
+* typically this file should be empty, but it depends on the ``<tool>``'s needs
 * even if empty, the file facillitates module importability and must be present
 
-3. Add a file named ``fre/<subtool>/fre<subtool>.py``. This will serve as the main entry point for ``fre``
-   into the ``<subtool>``'s functionality
+3. Add a file named ``fre/<tool>/fre<tool>.py``. This will serve as the main entry point for ``fre``
+   into the ``<tool>``'s functionality
 
-4. Add a ``click`` group named after ``<subtool>`` within ``fre/<subtool>/fre<subtool>.py``
+4. Add a ``click`` group named after ``<tool>`` within ``fre/<tool>/fre<tool>.py``
 
-* This ``click`` group will contain all the subcommands under the ``<subtool>``'s functionality
+* This ``click`` group will contain all the functionality under the ``<tool>``
 
-5. Create separate files as needed for different subcommands; do not code out the full
-   implemetation of ``<subtool>`` inside of a ``click`` command within ``fre/<subtool>/fre<subtool>.py``.
+5. Create separate files as needed for different commands; do not code out the full
+   implemetation of ``<tool>`` inside of a ``click`` command within ``fre/<tool>/fre<tool>.py``.
 
-* better yet, consider what structure your subtool may need in the future for maintainability's sake
+* better yet, consider what structure your tool may need in the future for maintainability's sake
+* if you need, specify a ``<subtool>`` like ``fre/<tool>/<subtool>``. ``fre/app`` currently has
+  this structure
 
-6. Be sure to import the contents of the needed subcommand scripts inside of ``fre<subtool>.py``
+6. Be sure to import the contents of the needed subcommand scripts inside of ``fre<tool>.py``
 
-* i.e. from ``fre.fre<subtool>.subCommandScript import *``
+* i.e. from ``fre.fre<tool>.toolCommandScript import *``
```

Comment:
Sorry to nit pick, one more clarifying thing. Before, I meant it should just be `<tool>` here not `fre<tool>` right? Because the structure is fre/make/createCheckout for example

Reply:
correct!